src/main/scala/com/actian/KafkaSparkVector/SimpleApp.scala

```diff
@@ -36,17 +42,13 @@ object SimpleApp {
     // Init Spark context
     val sparkConf  = new SparkConf().setAppName("Simple Application")
     val sc         = new SparkContext(sparkConf)
-    
-    
-
-
 
     // Init SQL context
     val sqlContext = new SQLContext(sc) 
     import sqlContext.implicits._
     
     // Change connection properties to fit your demo environment
```

Comment:
As the readme has been updated to remove the things to look at editing/customizing, recommend adding some easy search term (e.g. EDIT_ME) and mentioning in the readme too. Again not a show stopper.

Reply:
Agreed, will change that.